packages/account/src/Sections/Profile/PersonalDetails/__tests__/personal-details.spec.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { cleanup, render, waitFor, screen, fireEvent } from '@testing-library/react';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { PersonalDetailsForm } from '../personal-details-form';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import userEvent from '@testing-library/user-event';
+
+afterAll(cleanup);
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+
+jest.mock('@deriv/shared/src/services/ws-methods', () => ({
```

Comment:
Cant we just import `jest.mock('@deriv/shared,()=>({WS:......})`? What was the error thrown?